saltyrtc/client.ts

```diff
@@ -225,33 +197,42 @@ export class Client {
             }
 
             // Send offer
-            this.peerConnection.createOffer().then(
+            /*this.peerConnection.createOffer().then(
                 (offer) => {
                     this.peerConnection.setLocalDescription(offer); // TODO: Should we handle errors?
                     this._startConnectTimer();
                     this.signaling.sendOffer(offer);
                 },
                 (error) => console.error('PeerConnection error:', error)
-            );
-        });
-        this.$rootScope.$on('signaling:answer', (_, answer) => {
-            this.peerConnection.receiveAnswer(answer);
-        });
-        this.$rootScope.$on('signaling:candidate', (_, candidate) => {
-            this.peerConnection.receiveCandidate(candidate);
+            );*/
         });
+    }
 
-        // Listen for peer connection events and delegate them
-        this.$rootScope.$on('pc:candidate', (_, candidate) => {
-            this.signaling.sendCandidate(candidate);
-        });
+    /**
+     * Send an ICE candidate through the signalling channel.
+     *
+     * TODO: Make sure candidates are buffered for 10ms, according to the
+     * SaltyRTC spec.
+     */
+    public sendCandidate(candidate: RTCIceCandidate) {
```

Comment:
Right now we're only checking types against the WebRTC spec. ORTC is still very much in flux, right?

The RTCIceCandidate is defined [here](https://www.w3.org/TR/webrtc/#rtcicecandidate-interface). That definition matches the type spec committed into our repo.